### ISSUE TITLE
Is there a way to hide the code and show only formatted text? 

### ISSUE THREAD

**Issue by manu26c** [olivierkes/manuskript#959]
(ex, {C:1:John Smith} = **John Smith**

**Comment by TheJackiMonster** [comment]
No, currently there is no visual (you-see-is-what-you-get) editor for notes or even the text sections. But I think there's already an issue (#913) open for this feature.
